Added sync command to ucare cli

zerc opened the pull request:
Let's look at the direction of development this feature and make some discuss :)

Example of output :)

```
[##########] 100.00% 317976-2560x1600.jpg
[##########] 100.00% 476030-2560x1600.jpg
[##########] 100.00% 568399-2560x1600.jpg
[##########] 100.00% 517092-2560x1600.jpg
[##########] 100.00% 294836-2560x1600.jpg
[##########] 100.00% 13147160-R3L8T8D-650-KittenDogFriends.jpg
[##########] 100.00% 2015-02-20 15-35-51 iTunes.png
[##########] 100.00% 13157110-R3L8T8D-650-44433322211.jpg
```

dmitry-mukhin commented on pyuploadcare/ucare_cli.py:
```diff
@@ -145,6 +148,67 @@ def create_group(arg_namespace):
+            response.raise_for_status()
+        except requests.HTTPError as e:
+            pp.pprint(('Can\'t download file: `{}`. '
+                       'Origin exception: {}').format(url, e))
```
exception -> error

dmitry-mukhin commented on pyuploadcare/ucare_cli.py:
```diff
@@ -145,6 +148,67 @@ def create_group(arg_namespace):
+
+    for f in files:
+        url = f.info()['original_file_url']
+        local_filename = os.path.join(arg_namespace.path, f.filename())
```
you could provide a pattern to build file path, like during remote copying :)

![selection_040](https://cloud.githubusercontent.com/assets/925309/10286236/7cff0b90-6b95-11e5-9ddb-581276a747f4.png)

if that's too much, the easiest would be just to have UUID as part of the path

zerc replied:
I agree it could be useful

zerc commented:
I added support of this patterns keys:

```
${uuid}
${filename}
${effects}
${ext}
```

Default is `${uuid}${ext}` I think it will enough.

Also i added `--effects` param. It adding effects for downloaded files. I think it will be useful e.g. when you want get only previews of files.
